source/overloading.tex

```diff
@@ -3757,11 +3722,9 @@
 \indextext{postfix ++ and -{-} overloading@postfix \tcode{++} and \tcode{\dcr}!overloading}%
 
 \pnum
-The user-defined function called
-\tcode{operator++}
-implements the prefix and postfix
-\tcode{++}
-operator.
+An \defnadj{increment}{operator function}
+is a function named \tcode{\keyword{operator}++}
+that implements the prefix and postfix \tcode{++} operator.
```

Comment:
This last line is not part of the definition of "increment operator function". As written, this would suggest that an `operator++` that (somehow) doesn't implement a prefix and [or?] postfix `++` operator is not an "increment operator function".

Reply:
Maybe convert "that implements the prefix and postfix `++` operator" to "[Note: An increment operator function implements either the prefix or postfix `++` operator.]", or just remove it -- we say the same thing but better in the next couple of sentences.

Reply:
Removed.